Wi-Fi Zone/src/main/java/by/gsu/com/WiFiMetricsFileWiFiMetricsBuilderImpl.java

```diff
@@ -0,0 +1,54 @@
+package by.gsu.com;
+
+import by.gsu.com.Interface.WiFiMetricsBuilder;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+public class WiFiMetricsFileWiFiMetricsBuilderImpl implements WiFiMetricsBuilder {
```

Comment:
Please, change it's name to WiFiMetricsFilePropertiesWiFiMetricsBuilderImpl cause you work with properties here

Reply:
Fixed